contracts/governance/FeeSharingCollector/FeeSharingCollectorStorage.sol

```diff
@@ -100,6 +106,16 @@ contract FeeSharingCollectorStorage is Ownable {
         _;
         reentrancyLock = REENTRANCY_GUARD_FREE;
     }
+
+    /* Modifier */
+    modifier oneTimeExecution(bytes4 _funcSig) {
```

Comment:
How did it work before adding this? `oneTimeExecution` already existed as a modifier for multiple functions :D